### ISSUE TITLE
[Bug Report]: 

### ISSUE THREAD

**Issue by ramuvr** [Azure/ResourceModules#4150]
### Describe the bug

As per [MS Doc](https://learn.microsoft.com/en-us/azure/application-gateway/proxy-buffers), the value for buffers  should default to true.

"By default, both Request and Response buffers are enabled on your Application Gateway resource but you can choose to configure them separately."

### To reproduce

Access file modules\network\application-gateway\main.bicep and see code snippet values

### Code snippet

```Bicep
@description('Optional. Enable request buffering.')
param enableRequestBuffering bool = false

@description('Optional. Enable response buffering.')
param enableResponseBuffering bool = false
```

**Comment by AlexanderSehr** [comment]
Hey @ramuvr , thank you for opening this back and thank you for providing the docs reference.
If I understand you correctly, all this takes is changing both buffering switches to `true` to align with the best-practices? If so, we can certainly do that.

**Comment by ramuvr** [comment]
> Hey @ramuvr , thank you for opening this back and thank you for providing the docs reference. If I understand you correctly, all this takes is changing both buffering switches to `true` to align with the best-practices? If so, we can certainly do that.

Yes, @AlexanderSehr  that's my understanding. Deployed couple of appGateways recently without the attribute and noticed they were false but our template had default value so what-if was reporting I was going to change them.
